Library/Formula/unrar.rb

```diff
@@ -15,6 +15,11 @@ class Unrar < Formula
   def install
     system "make"
     bin.install 'unrar'
+    system "make", "clean"
```

Comment:
Unfortunately, this is out of my expertise.  I will write them a note describing the issues.  In the meantime, I hope will you accept my current changes to this formula which are both simple and functional.  Thanks!